src/lib/media-import/status.js

```diff
@@ -235,11 +239,22 @@ ${ maybeExitPrompt }
 				}
 				overallStatus = status;
 
-				setTimeout( checkStatus, IMPORT_MEDIA_PROGRESS_POLL_INTERVAL );
+				// after two minutes, we'll start decreasing the pollInterval
+				pollIntervalDecreasing = startDate < ( Date.now() - TWO_MINUTES_IN_MILLISECONDS );
+
+				// decrease poll interval by a second, every minute
+				if ( pollIntervalDecreasing && startDate < ( Date.now() - ONE_MINUTE_IN_MILLISECONDS ) ) {
+					pollInterval = pollInterval + IMPORT_MEDIA_PROGRESS_POLL_INTERVAL;
+					startDate = Date.now();
```

Comment:
Haven't tested this out but just going on the basis of a dry run.. but wouldn't this reset the `startDate` for the two minute mark check?

Reply:
I had an IF first and did some refactoring, but forgot to add the condition for that. Fixed now, thanks!

Reply:
ah okay thanks!